examples/with-supertokens/package.json

```diff
@@ -6,10 +6,14 @@
     "start": "next start"
   },
   "dependencies": {
+    "@babel/core": "^7.18.13",
```

Comment:
Why was `@babel/core` added? This shouldn't be needed. Also looks like `@types/node` and `@types/react` are missing from `devDependencies`.

Reply:
`@babel/core` is a peer dependency of some of the packages added for both supertokens and some other stuff. I can remove it if you would like, I was just trying to resolve the dependency issue returned using `pnpm`. I am adding the two type packages along with some slight code changes. Just let me know what you think about `@babel/core`.

Reply:
I see, yeah lets drop `@babel/core` as it shouldn't be needed and it sounds like there's an invalid `peerDependency` requirement somewhere in the tree so fixing that upstream would be more ideal.

Reply:
That makes sense. My most recent commit removes that package from dependencies.